core/tests/recovery-test/tests/snapshot-recovery.test.ts

```diff
@@ -182,23 +209,28 @@ describe('snapshot recovery', () => {
     });
 
     step('drop external node database', async () => {
-        await dropNodeDatabase(externalNodeEnv);
+        await dropNodeDatabase(fileConfig.loadFromFile, externalNodeEnv);
     });
 
     step('drop external node storage', async () => {
-        await dropNodeStorage(externalNodeEnv);
+        await dropNodeStorage(fileConfig.loadFromFile, externalNodeEnv);
     });
```

Comment:
let's unify this commands

Reply:
Addressed in https://github.com/matter-labs/zksync-era/pull/2444/commits/46c2d5515a82071b860e818c5f3ac0ba05a3a709